docs/advanced/debugging.rst

```diff
@@ -5,14 +5,27 @@ Debugging
 Logging
 =======
 
-The Flocker processes running on the nodes will write their logs to ``/var/log/flocker/``.
-The log files are named ``<processname>-<pid>.log``, e.g. ``flocker-volume-1234.log``.
+Flocker processes generally use `eliot`_ for logging.
+These logs can be rendered as an ASCII tree using `eliottree`_.
+
+Ubuntu
+^^^^^^
+
+XXX This should be documented, see :issue:`1877`.
+
+Fedora / CentOS
+^^^^^^^^^^^^^^^
+
+Logs from the Flocker processes running on the nodes are written to `systemd's journal`_.
+They have unit names starting constructed with a ``flocker-`` prefix, e.g. ``flocker-agent``.
 
 Logs from the Docker containers are written to `systemd's journal`_ with a unit name constructed with a ``ctr-`` prefix.
-For example if you've started an application called ``mymongodb`` you can view its logs by running the following command on the node where the application was started:
+For example if you have started an application called ``mymongodb`` you can view its logs by running the following command on the node where the application was started:
```

Comment:
What *is* accessible from journald is flocker logs: `journalctl -u flocker-agent`, `journalctl -u flocker-control` and `journalctl -u flocker-container-agent`.